src/main/java/org/aludratest/config/impl/SimplePreferences.java

```diff
@@ -29,11 +29,16 @@
  */
 public class SimplePreferences extends AbstractPreferences implements MutablePreferences {
 
-    private Map<String, String> values = new HashMap<String, String>();
+    private Map<String, String> values;
```

Comment:
Anyway I consider this good coding style to initialize all non-static attributes only in the constructor (which is BTW my default approach): You have all initializations in a single place, do not get messed up when changing the order of the attributes and other users are less likely to misunderstand trickier parts of the initialization. OK, for a single attribute this doesn't matter a lot, but I would adhere to the rule anyway.